test/initial_release/GHTestServer.d

```diff
@@ -0,0 +1,200 @@
+/*******************************************************************************
+
+    Github Test Server class
+
+    Copyright:
+        Copyright (c) 2017 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module test.initial_release.GHTestServer;
```

Comment:
It might be a good idea to move to makd v2 or at least use `integrationtest` as the directory to store integration tests instead.